BUILD.md

```diff
@@ -60,6 +60,11 @@ The complete list of build options can be found by running `./build.sh (or ./bui
 1. For Windows, just add --x86 argument when launching build.bat
 2. For Linux, it must be built out of a x86 os, --x86 argument also needs be specified to build.sh
 
+## Build Hosting Application on Linux
+
+1. Besides ONNX Runtime dependences, `g++-7` or greater is needed to build vcpkg.
```

Comment:
Do you mean we will not support ubuntu 16?

Reply:
No, we do. Our current VSTS build machines are on 16.04

Reply:
It's not required for building hosting -- it's required for building vcpkg. So no to your second question.

But this may all be moot. I have a PR in Klein's fork to remove vcpkg and install Boost from source to the binary folder if building hosting: https://github.com/NonStatic2014/onnxruntime/pull/39